accounts/abi/bind/auth.go

```diff
@@ -130,14 +132,15 @@ func NewKeyStoreTransactorWithChainID(keystore *keystore.KeyStore, account accou
 	if chainID == nil {
 		return nil, ErrNoChainID
 	}
-	var signer types.Signer = types.NewEIP155Signer(chainID)
+	var Signer types.Signer = types.NewEIP155Signer(chainID)
```

Comment:
why capital `Signer` variable name?

Reply:
it's for differentiating from the signer l.143. It can't be here as l.145 will override this default value.

Reply:
can we use another variable name more explicit instead of using capital S (maybe `eipo155Signer`)? as that confuses with public struct or interface names.